src/EnergyPlus/CurveManager.hh

```diff
@@ -334,15 +336,12 @@ namespace Curve {
 
     int GetCurveIndex(EnergyPlusData &state, std::string const &CurveName); // name of the curve
 
+    Curve *GetCurve(EnergyPlusData &state, std::string const &curveName);
+
+    Curve *AddCurve(EnergyPlusData &state, std::string const &curveName);
+  
     // This utility function grabs a curve index and performs the
     // error checking
-
```

Comment:
Not used anymore (was only used in two modules). Not really a fan of this pattern.